lib/saluki-event/src/eventd/priority.rs

```diff
@@ -0,0 +1,9 @@
+/// EventPriority is the event priority for events
```

Comment:
Similar comment here as the one on `ServiceCheckStatus` about avoiding stuttering, etc.